### DISCUSSION TITLE
Grouped notifications

### DISCUSSION THREAD

**Discussion by Wildheart-Baby** [tuskyapp/Tusky#3229] [Ideas]
I was wondering if there were plans to have notifications be grouped, it would be nice to have one notification on a post, and how many boosted or liked the post, rather then seeing every boost or like, as a long list.

**Comment by koehntopp** [comment]
The notifications are driving me crazy. It would really help if I could effectively limit them ("just mnentions"), and if they were grouped instead of filling up my whole screen during an active discussion.

**Comment by connyduck** [comment]
1) yes there are plans, see https://github.com/tuskyapp/Tusky/issues/4715

2) you can already filter notifications by type
